src/main/java/Barney.java

```diff
@@ -17,18 +18,28 @@ public static void welcomeText() {
     }
 
     public static void chat() {
-        String input = "";
+        String input;
         Boolean isChatting = true;
```

Comment:
Good naming of boolean variable that follows the coding standard!

Reply:
Thank you!